lib/org/chromium/webidl/ast/Implements.js

```diff
@@ -8,6 +8,8 @@ foam.CLASS({
   name: 'Implements',
   extends: 'org.chromium.webidl.ast.DefinitionData',
 
+  ids: [ 'implementer', 'implemented' ],
```

Comment:
This comment applies to all `ids` fields. Is there any performance concerns that we have to worry about? `implementer` and `implemented` are objects (with a `literal` property that is a string), and I'm not certain if the usage of objects as ID will be optimal.